server/src/routes/admin/customer.ts

```diff
@@ -1,39 +1,91 @@
-import { sql } from "@/libs/db";
-import Elysia, { t } from "elysia";
-import { NewCustomerSchema } from "@/libs/validation";
+import { sql } from '@/libs/db';
+import Elysia, { t } from 'elysia';
+import { NewCustomerSchema } from '@/libs/validation';
 
-export const customerRoutes = new Elysia({ prefix: "/customers" })
-  .post('/', async ({ body, set }) => {
-    const validation = NewCustomerSchema.safeParse(body);
+export const customerRoutes = new Elysia({ prefix: '/customers' })
+    .post(
+        '/',
+        async ({ body, set }) => {
+            const validation = NewCustomerSchema.safeParse(body);
 
-    if (!validation.success) {
-      set.status = 400;
+            if (!validation.success) {
+                set.status = 400;
 
-      return {
-        status : 400,
-        message : validation.error.errors[0].message
-      }
-    }
+                return {
+                    status: 400,
+                    message: validation.error.errors[0].message,
+                };
+            }
 
-    const { first_name, last_name, address, sub_district_id, district_id, province_id, postal_code, phone_number, email } = body;
-    
-    await sql`INSERT INTO customers_detail(first_name, last_name, address, sub_district_id, district_id, province_id, postal_code, phone_number, email )
-              VALUES (${first_name}, ${last_name}, ${address}, ${sub_district_id}, ${district_id}, ${province_id}, ${postal_code}, ${phone_number}, ${email})`;
-    return {
-      status : "success",
-      message : "customer added successfully."
-    }
+            const {
+                first_name,
+                last_name,
+                address,
+                sub_district_id,
+                district_id,
+                province_id,
+                postal_code,
+                phone_number,
+                email,
+            } = body;
 
-  }, {
-    body: t.Object({
-      first_name: t.String(),
-      last_name: t.String(),
-      address: t.String(),
-      sub_district_id: t.String(),
-      district_id: t.String(),
-      province_id: t.String(),
-      postal_code: t.String(),
-      phone_number: t.String(),
-      email: t.String(),
-    })
-  });
\ No newline at end of file
+            await sql`INSERT INTO customers_detail(first_name, last_name, address, sub_district_id, district_id, province_id, postal_code, phone_number, email )
+              VALUES (${first_name}, ${last_name}, ${address}, ${sub_district_id}, ${district_id}, ${province_id}, ${postal_code}, ${phone_number}, ${email})`;
+            return {
+                status: 'success',
+                message: 'customer added successfully.',
+            };
+        },
+        {
+            body: t.Object({
```

Comment:
can remove
```
{
            body: t.Object({
                first_name: t.String(),
                last_name: t.String(),
                address: t.String(),
                sub_district_id: t.String(),
                district_id: t.String(),
                province_id: t.String(),
                postal_code: t.String(),
                phone_number: t.String(),
                email: t.String(),
            }),
        }
```
because can validate data by newCustomerSchema

Reply:
removed